lmdk/libraries/module_example/module_example_mtl.toml

```diff
@@ -0,0 +1,86 @@
+version = [3, 0]
```

Comment:
why is this file needed, and particularly in this PR?

Reply:
toml is modular now, so we can take the up/down mixer parts in the correct place.